docs/content/en/partials/_index-data-access-right.md

```diff
@@ -0,0 +1,10 @@
+
+## Data Access
+
+### standardizing access for all Observability Data for K8s
```

Comment:
```suggestion
### Standardizing access for all Observability Data for K8s
```

Reply:
Do we even need this ### header? In the rendered template is the same as the following line

Reply:
there is a slight difference and it also offers some kind of syntactical difference, if we decide to eg. style it differently in the future.